index.rst

```diff
@@ -52,11 +52,11 @@ In this documentation
 Project and community
 ---------------------
 
-Example Project is a member of the Ubuntu family. It’s an open source project that warmly welcomes community projects, contributions, suggestions, fixes and constructive feedback.
+JAAS is a member of the Ubuntu family. It’s an open source project that warmly welcomes community projects, contributions, suggestions, fixes and constructive feedback.
```

Comment:
I think this line probably needs to be changed from the default because JAAS isn't open-source.

Reply:
@alesstimec has already fixed this paragraph in his PR. I'll just merge with the main.